runtime-light/tl/tl-functions.cpp

```diff
@@ -0,0 +1,48 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime-light/tl/tl-functions.h"
+
+#include <cstdint>
+#include <string_view>
+
+#include "runtime-light/tl/tl-core.h"
+
+namespace {
+
+// magic + flags + image_id + job_id + minimum string size length
+constexpr auto K2_INVOKE_JW_MIN_SIZE = 2 * sizeof(uint32_t) + sizeof(uint64_t) + sizeof(int64_t) + tl::SMALL_STRING_SIZE_LEN;
```

Comment:
I suppose `sizeof(K2_INVOKE_JOB_WORKER_MAGIC) + sizeof(K2InvokeJobWorker::image_id)` is more expressive.
Imagine once you change the type of structure, `K2_INVOKE_JW_MIN_SIZE` will be recalculated automatically

Reply:
I disagree. We shouldn't mix TL-types like `uint32_t`, `int64_t` and their c++ representation in particular case.